packages/synapse-react-client/src/utils/functions/EntityTypeUtils.ts

```diff
@@ -126,41 +126,32 @@ export function entityTypeToFriendlyName(entityType: EntityType): string {
 export function convertToEntityType(
   typeString: string | ENTITY_CONCRETE_TYPE | EntityType,
 ): EntityType {
+  if (Object.values(EntityType).includes(typeString as EntityType)) {
+    return typeString as EntityType
+  }
```

Comment:
Changes in this file were also flagged by [@typescript-eslint/no-unsafe-enum-comparison](https://typescript-eslint.io/rules/no-unsafe-enum-comparison/). It didn't like that we were comparing `typeString` (a string) to `EntityType` via `switch`.

Since the goal of this function is to return an `EntityType`, change the logic to see if `typeString` is an entity type first, then do the comparison. This function is well-tested, so this should be a safe change.